ddp-workspace/projects/ddp-dsm-ui/src/app/navigation/navigation.component.ts

```diff
@@ -48,8 +48,8 @@ export class NavigationComponent implements OnInit {
 
   }
 
-  public showOncHistoryUploadPage(selectedStudy: string | null): boolean {
-    return selectedStudy && this.oncHistoryUploadPageAllowedStudies.includes(selectedStudy);
+  public showPecgsFeatures(selectedStudy: string | null): boolean {
```

Comment:
This looks like it's more of a "should" than a "do it", so naming it `shouldShowPecgs...()` will help readers understand that this is checking auth, not actually showing the pecgs features.